src/clientManager.js

```diff
@@ -32,12 +32,29 @@ ClientManager.prototype.findClient = function(name) {
 };
 
 ClientManager.prototype.autoloadUsers = function() {
-	this.getUsers().forEach((name) => this.loadUser(name));
+	const users = this.getUsers();
+	const noUsersWarning = `There are currently no users. Create one with ${colors.bold("lounge add <name>")}.`;
+
+	// There was an error, already logged, but we have to crash the server as
+	// user directory could not be accessed
+	if (users === undefined) {
+		process.exit(1);
+	}
+
+	if (!users.length) {
+		log.warn(noUsersWarning);
+	}
+
+	users.forEach((name) => this.loadUser(name));
 
 	fs.watch(Helper.USERS_PATH, _.debounce(() => {
 		const loaded = this.clients.map((c) => c.name);
 		const updatedUsers = this.getUsers();
 
+		if (!updatedUsers.length) {
```

Comment:
Alright, info is fine. Made it info consistently to help admins search through their logs or something.